manifests/charts/base/templates/services.yaml

```diff
@@ -24,5 +24,11 @@ spec:
   type: ExternalName
   externalName: {{ .Values.global.remotePilotAddress }}
   {{- end }}
+  {{- if .Values.global.ipFamilyPolicy }}
+  ipFamilyPolicy: {{ .Values.global.ipFamilyPolicy }}
+  {{- end }}
+  {{- with .Values.global.ipFamilies }}
```

Comment:
do we want to have an if check here as well?

Reply:
I've added a comment that is largely intended for use with external istiod. I haven't ever installed external istiod (or even multicluster) with just the helm charts. Do you think I should remove? That may be another issue to track onto itself.